core/src/main/java/io/questdb/cairo/TxReader.java

```diff
@@ -355,29 +382,30 @@ public long unsafeReadVersion() {
 
     private void clearData() {
         baseOffset = 0;
-        size = 0;
-        partitionTableVersion = -1;
+        size = 0L;
+        partitionTableVersion = -1L;
```

Comment:
Is it a code inspection rule? Can you share it with the team so that we all on the same coding style?